Tests/SourceKitLSPAPITests/SourceKitLSPAPITests.swift

```diff
@@ -255,6 +255,70 @@ final class SourceKitLSPAPITests: XCTestCase {
             ]
         )
     }
+
+    func testLoadPackage() async throws {
+        let fs = InMemoryFileSystem(emptyFiles:
+            "/Pkg/Sources/lib/lib.swift"
+        )
+
+        let observability = ObservabilitySystem.makeForTesting()
+        let graph = try loadModulesGraph(
+            fileSystem: fs,
+            manifests: [
+                Manifest.createRootManifest(
+                    displayName: "Pkg",
+                    path: "/Pkg",
+                    toolsVersion: .v5_10,
+                    targets: [
+                        TargetDescription(
+                            name: "lib",
+                            dependencies: []
+                        )
+                    ]),
+            ],
+            observabilityScope: observability.topScope
+        )
+        XCTAssertNoDiagnostics(observability.diagnostics)
+
+        let destinationBuildParameters = mockBuildParameters(destination: .target)
+        try await withTemporaryDirectory { tmpDir in
+            let pluginConfiguration = PluginConfiguration(
+                scriptRunner: DefaultPluginScriptRunner(
+                    fileSystem: fs,
+                    cacheDir: tmpDir.appending("cache"),
+                    toolchain: try UserToolchain.default
+                ),
+                workDirectory: tmpDir.appending("work"),
+                disableSandbox: false
+            )
+            let scratchDirectory = tmpDir.appending(".build")
+
+            let loaded = try await BuildDescription.load(
```

Comment:
question: Is there value in adding additional tests that will validate the `BuildDescription.load`  behaves as expected considering different destination Build Parameter, different graph, filesystem layout, scratch directory?

Reply:
It is just calling into `BuildOperation`, which we have a billion other tests for. So no, I don't think there's particular value in that here.